schemas/webhooktunnel-token-response.yml

```diff
@@ -0,0 +1,20 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:                      "Webhooktunnel Token Response"
+description: |
+    Token for connecting a worker to webhooktunnel proxy
+type:                       object
+properties:
+  id:
+    type: string
+    title: Proxy Connection ID
+    description: |
+      id for proxy connection
+  token:
+    type: string
+    title: Auth Token
+    description: |
```

Comment:
sure, it could cause issues elsewhere...

Actually, you're perfectly right that it's not necessary... you can leave it out if you want to. In fact it might be overkill to add a regex for jwts... I just generally to be as explicit as reasonably possible in the JSON schemas, as it catches a lot of bugs..